pontoon/base/models.py

```diff
@@ -25,12 +25,72 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
+    """
+    Provides various method to interact with larger sets of translations and their stats for user.
+    """
+
+    def _changed_translations_count(self, args):
+        """
+        Helper method, returns expression object which allows us to annotate querysets
+        with counts of translations.
+        """
+        translation_query = (
+            ~Q(translation__string=F('translation__entity__string'))
+            & ~Q(translation__string=F('translation__entity__string_plural'))
+            & Q(translation__user__isnull=False)
```

Comment:
wat

That's really odd. I guess leave it in for now, and we can take a closer look later. If this works as expected and the tests pass, it probably doesn't have any side effects. So either I'm wrong (very very possible) or something else is causing the issue and this just happens to fix it. Either possibility can be resolved later.